test/demo/contextListener.test.ts

```diff
@@ -1,48 +1,135 @@
 import 'jest';
-import * as fdc3Remote from './utils/fdc3RemoteExecution';
-import {getFinConnection} from './utils/fin';
+import {connect, Fin} from 'hadouken-js-adapter';
+
+import {OrganizationContext} from '../../src/client/main';
 
+import * as fdc3Remote from './utils/fdc3RemoteExecution';
 
-describe('advanced puppeteer functionality', () => {
-    const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
-    const testIdentity1 = {uuid: 'test-app-1', name: 'test-app-1'};
-    const testIdentity2 = {uuid: 'test-app-2', name: 'test-app-2'};
+const testManagerIdentity = {
+    uuid: 'test-app',
+    name: 'test-app'
+};
 
-    it('can can register and trigger a context listener', async () => {
-        // Creating the apps can take a while, so we need a longer timeout.
-        jest.setTimeout(20000);
+const validContext: OrganizationContext = {type: 'organization', name: 'OpenFin', id: {default: 'openfin'}};
 
-        const fin = await getFinConnection();
+// These tests all use the default channel for context broadcasts and listeners
+// Context-passing in a channelled environment is tested in a seperate file
+describe('Context listeners and broadcasting', () => {
+    // Adapter connection is established in setup. Reassign here for convenience
+    const fin: Fin = global.__FIN__;
```

Comment:
Is it possible to make this a util/import, same as other dependencies? Seems the env script runs early enough for that to happen?..

Reply:
Done